mlir/include/mlir/Interfaces/LoopLikeInterface.td

```diff
@@ -218,6 +221,53 @@ def LoopLikeOpInterface : OpInterface<"LoopLikeOpInterface"> {
         return ::mlir::failure();
       }]
     >,
+    InterfaceMethod<[{
+        Append the specified additional "init" operands: replace this loop with
+        a new loop that has the additional init operands. The loop body of
+        this loop is moved over to the new loop.
+
+        This method is similar to `replaceWithAdditionalYields` but instead of
+        returning the value that is actually yielded, this returns the tiles of
```

Comment:
This does not seem to correctly describe the method: it returns a LoopLikeOpInterface but here you're saying "this returns the tiles of the values that are yielded"?